### ISSUE TITLE
bioconda dependencies

### ISSUE THREAD

**Issue by pmenzel** [phac-nml/staramr#133]
After fighting for a while with installing staramr via bioconda, I figured out that the latest versions of blast and pandas cause issues with staramr.

It worked after manually setting the environment like this:
```
dependencies:
  - pandas=1.1.5
  - bioconda::staramr=0.7.2
  - bioconda::blast=2.9.0
```

So maybe this needs to be reflected in https://github.com/bioconda/bioconda-recipes/blob/master/recipes/staramr/meta.yaml  or staramr needs an update to work with latest versions of these tools.

**Comment by apetkau** [comment]
Thank you for bringing this up. I will have to look into what's going on with these dependencies.

**Comment by apetkau** [comment]
This was fixed in the bioconda recipe for release `0.7.2` a little while ago. The restriction on the version of `pandas` has also been fixed in PR #146
